src/home/Home.js

```diff
@@ -33,7 +59,23 @@ const styles = StyleSheet.create({
     fontSize: 25,
     marginHorizontal: 30,
     letterSpacing: 0.3,
-    width: '100%'
+    width: '100%',
+    paddingBottom: 11
+  },
+  subtitle: {
+    color: '#453E3D',
+    fontWeight: '400',
+    fontSize: 16,
+    fontStyle: 'italic',
+    marginLeft: 30,
+    letterSpacing: 0.3,
+    width: '55%',
+    paddingBottom: 4
+  },
```

Comment:
Does hard coding margins affect how the display looks on a different sized screen? I am just asking because it seems to done here and there in your code